security/wildfly/pom.xml

```diff
@@ -45,7 +44,7 @@
         <!-- Provisioning properties -->
         <feature.pack.groupId>org.wildfly</feature.pack.groupId>
         <feature.pack.artifactId>wildfly-ee-galleon-pack</feature.pack.artifactId>
-        <feature.pack.version>${version.org.wildfly}</feature.pack.version>
+        <feature.pack.version/>
```

Comment:
Was this deliberate?

Reply:
Yes. To always use the latest version of WildFly. If preferred I could use a hard-coded version.